frontend/src/Dashboard/CrossingListPage/StatusToggle.scss

```diff
@@ -0,0 +1,86 @@
+@import "../../css/flexbox";
+@import "../../css/font-awesome/font-awesome.scss";
+
+$container-height: 46px;
+
+$border-color: #D0CED7;
+$text-color: #D0CED7;
+$selected-text-color: #11022F;
```

Comment:
Wondering if these colors would be used more globally. And if maybe we need a colors sass file with all the colors we use listed as variables. We could punt on this but something to keep in mind.